lib/thunderbird_common.pm

```diff
@@ -173,7 +191,7 @@ sub tb_check_email {
     wait_screen_change { send_key "shift-f5" };
     send_key "ctrl-shift-k";
     wait_screen_change { type_string "$mail_search" };
-    assert_screen "thunderbird_sent-message-received";
+    assert_screen("thunderbird_sent-message-received", timeout => 120);
```

Comment:
better be consistent, just write `assert_screen 'thunderbird_sent-message-received', 120;`

Reply:
ok, done